tools/commands.py

```diff
@@ -46,23 +52,66 @@ def data_collector(config, catch=True):
 #             raise e
 
 
-def package_manager(config, commands, catch=False, force=False, default_git_branch=TENTACLES_DEFAULT_BRANCH):
+def _check_arg_and_cleanup(command_args, argument):
+    if argument in command_args:
+        command_args.pop(command_args.index(argument))
+        return True
+    return False
+
+
+async def _handle_package_manager_command(command_args, aiohttp_session=None):
+    aiohttp_session = aiohttp_session or aiohttp.ClientSession()
+    force = _check_arg_and_cleanup(command_args, FORCE_ARG)
+    all = _check_arg_and_cleanup(command_args, ALL_ARG)
+    install = _check_arg_and_cleanup(command_args, INSTALL_ARG)
+    update = _check_arg_and_cleanup(command_args, UPDATE_ARG)
+    uninstall = _check_arg_and_cleanup(command_args, UNINSTALL_ARG)
+
+    if not (all or command_args):
+        get_logger(COMMANDS_LOGGER_NAME).error("Please provide at least one tentacle name or add the 'all' parameter")
+    elif install:
+        if all:
+            await install_all_tentacles(DEFAULT_TENTACLES_URL, aiohttp_session=aiohttp_session)
+        else:
+            await install_tentacles(command_args, DEFAULT_TENTACLES_URL, aiohttp_session=aiohttp_session)
+    elif update:
+        if all:
+            await update_all_tentacles(DEFAULT_TENTACLES_URL, aiohttp_session=aiohttp_session)
+        else:
+            await update_tentacles(command_args, DEFAULT_TENTACLES_URL, aiohttp_session=aiohttp_session)
+    elif uninstall:
+        if all:
+            await uninstall_all_tentacles(DEFAULT_TENTACLES_URL, use_confirm_prompt=force)
+        else:
+            await uninstall_tentacles(command_args, DEFAULT_TENTACLES_URL, use_confirm_prompt=force)
+
+
+def _print_tentacle_manager_help():
```

Comment:
I think this should be defined in a tentacle manager entrypoint

Reply:
Same as the next answer, it depends if we want to share the command line responsibility with the tentacle manager or not. I think it's better to expose the API and leave the choice to the caller to use a CLI or not and how the CLI should behave